code/upaya/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './public/js/entry.js',
+
+  output: {
+    filename: 'bundle.js',
+    path: __dirname + '/public/js'
```

Comment:
yep, was going to move to that once we actually had some real js in here, but happy to start now.